package.json

```diff
@@ -9,9 +9,9 @@
   "type": "module",
   "scripts": {
     "test": "standard index.js",
-    "publish": "npm publish && npm publish --registry https://npm.pkg.github.com",
+    "pub": "npm pub && npm publish --registry https://npm.pkg.github.com",
```

Comment:
Should be `npm run pub` right?

Reply:
Or wait what does `npm pub` do

Reply:
it's an alias for `npm publish`. I just copied this line from the `io`'s package.json